docs/tutorials/autogluon-cloud.md

```diff
@@ -1,108 +1,184 @@
+Costs for running cloud compute are managed by Amazon SageMaker, and storage costs are managed by AWS S3. AutoGluon-Cloud is a wrapper to these services at no additional charge. While AutoGluon-Cloud makes an effort to simplify the usage of these services, it is ultimately the user's responsibility to monitor compute usage within their account to avoid unexpected charges.
 ```
 
 
 ## Installation
 `autogluon.cloud` does not come with the default `autogluon` installation. You can install it via:
 
 ```bash
-pip3 install autogluon.cloud
+pip install autogluon.cloud
 ```
 
 Also ensure that the latest version of sagemaker python API is installed via:
 
 ```bash
-pip3 install -U sagemaker
+pip install -U sagemaker
 ```
 
 This is required to ensure the information about newly released containers is available.
 
-## Prepare an AWS Role with Necessary Permissions
-`autogluon.cloud` utilizes various AWS resources to operate.
-To help you to setup the necessary permissions, you can generate trust relationship and IAM policy with our utils through
+## Prepare an IAM Role with Necessary Permissions
+Currently, AutoGluon-Cloud can use two cloud backends: **Amazon SageMaker** and **Ray (AWS)**.
+Here is an overview of the features supported by each backend.
 
-```python
-from autogluon.cloud import TabularCloudPredictor  # Can be other CloudPredictor as well
+| Feature                        | SageMaker | Ray (AWS) |
+|--------------------------------|---------------|--------------|
+| **Supported modalities**       | `tabular`, `timeseries`, `multimodal` | `tabular` |
+| **Training (single instance)** | ✅ | ✅ |
+| **Training (distributed)**     | ❌ | ✅ |
+| **Inference endpoints**        | ✅ | ❌ |
+| **Batch inference**            | ✅ | ❌ |
 
-TabularCloudPredictor.generate_default_permission(
-    backend="BACKNED_YOU_WANT",  # We currently support "sagemaker" and "ray_aws"
-    account_id="YOUR_ACCOUNT_ID",  # The AWS account ID you plan to use for CloudPredictor.
-    cloud_output_bucket="S3_BUCKET"  # S3 bucket name where intermediate artifacts will be uploaded and trained models should be saved. You need to create this bucket beforehand.
-)
-```
+AutoGluon-Cloud needs to interact with various AWS resources. For this purpose, we recommend to set up a dedicated IAM role with the necessary permissions. This can be done using one of the following options.
 
-```{note}
-Make sure you review the trust relationship and IAM policy files, and make necessary changes according to your use case before applying them.
-```
+::::{tab-set}
+:::{tab-item} CloudFormation (AWS CLI)
+:sync: setup-cli
+1. Download and review the CloudFormation template from the [AutoGluon-Cloud repository](https://github.com/autogluon/autogluon-cloud/tree/add-cfn-templates/cloudformation)
```

Comment:
Links should be updated to `https://github.com/autogluon/autogluon-cloud/tree/master/cloudformation` after the PR is merged.